packages/packages/commonwell-cert-runner/src/document-contribution.ts

```diff
@@ -130,5 +154,356 @@ async function getOrCreateOrg(
iNXZWlnaHRfMSIvPg0KCQkJCQkJCQkJPC90ZXh0Pg0KCQkJCQkJCQkJPHN0YXR1c0NvZGUgY29kZT0iY29tcGxldGVkIi8+DQoJCQkJCQkJCQk8ZWZmZWN0aXZlVGltZSB2YWx1ZT0iMjAxNDEwMDExMDMwMjYtMDUwMCIvPg0KCQkJCQkJCQkJPHZhbHVlIHhzaTp0eXBlPSJQUSIgdmFsdWU9IjEwOC44NjMiIHVuaXQ9ImtnIi8+DQoJCQkJCQkJCQk8YXV0aG9yIHR5cGVDb2RlPSJBVVQiPg0KCQkJCQkJCQkJCTx0aW1lIHZhbHVlPSIyMDE0MTAwMTEwMzAyNi0wNTAwIi8+DQoJCQkJCQkJCQkJPGFzc2lnbmVkQXV0aG9yPg0KCQkJCQkJCQkJCQk8aWQgZXh0ZW5zaW9uPSI1NTU1NTU1NTU1IiByb290PSIxLjEuMS4xLjEuMS4xLjIiLz4NCgkJCQkJCQkJCQk8L2Fzc2lnbmVkQXV0aG9yPg0KCQkJCQkJCQkJPC9hdXRob3I+DQoJCQkJCQkJCTwvb2JzZXJ2YXRpb24+DQoJCQkJCQkJPC9jb21wb25lbnQ+DQoJCQkJCQkJPGNvbXBvbmVudD4NCgkJCQkJCQkJPG9ic2VydmF0aW9uIGNsYXNzQ29kZT0iT0JTIiBtb29kQ29kZT0iRVZOIj4NCgkJCQkJCQkJCTx0ZW1wbGF0ZUlkIHJvb3Q9IjIuMTYuODQwLjEuMTEzODgzLjEwLjIwLjIyLjQuMjciIGV4dGVuc2lvbj0iMjAxNC0wNi0wOSIvPg0KCQkJCQkJCQkJPGlkIHJvb3Q9IjU4NThlNzY1LTJmZmUtNDEzZi05MTk3LTI2MGYyYzZlN2FhOCIvPg0KCQkJCQkJCQkJPGNvZGUgY29kZT0iMzkxNTYtNSIgY29kZVN5c3RlbT0iMi4xNi44NDAuMS4xMTM4ODMuNi4xIiBjb2RlU3lzdGVtTmFtZT0iTE9JTkMiIGRpc3BsYXlOYW1lPSJCT0RZIE1BU1MgSU5ERVgiLz4NCgkJCQkJCQkJCTx0ZXh0Pg0KCQkJCQkJCQkJCTxyZWZlcmVuY2UgdmFsdWU9IiNCTUlfMSIvPg0KCQkJCQkJCQkJPC90ZXh0Pg0KCQkJCQkJCQkJPHN0YXR1c0NvZGUgY29kZT0iY29tcGxldGVkIi8+DQoJCQkJCQkJCQk8ZWZmZWN0aXZlVGltZSB2YWx1ZT0iMjAxNDEwMDExMDMwMjYtMDUwMCIvPg0KCQkJCQkJCQkJPHZhbHVlIHhzaTp0eXBlPSJQUSIgdmFsdWU9IjM3LjU4IiB1bml0PSJrZy9tMiIvPg0KCQkJCQkJCQkJPGF1dGhvciB0eXBlQ29kZT0iQVVUIj4NCgkJCQkJCQkJCQk8dGltZSB2YWx1ZT0iMjAxNDEwMDExMDMwMjYtMDUwMCIvPg0KCQkJCQkJCQkJCTxhc3NpZ25lZEF1dGhvcj4NCgkJCQkJCQkJCQkJPGlkIGV4dGVuc2lvbj0iNTU1NTU1NTU1NSIgcm9vdD0iMS4xLjEuMS4xLjEuMS4yIi8+DQoJCQkJCQkJCQkJPC9hc3NpZ25lZEF1dGhvcj4NCgkJCQkJCQkJCTwvYXV0aG9yPg0KCQkJCQkJCQk8L29ic2VydmF0aW9uPg0KCQkJCQkJCTwvY29tcG9uZW50Pg0KCQkJCQkJCTxjb21wb25lbnQ+DQoJCQkJCQkJCTxvYnNlcnZhdGlvbiBjbGFzc0NvZGU9Ik9CUyIgbW9vZENvZGU9IkVWTiI+DQoJCQkJCQkJCQk8dGVtcGxhdGVJZCByb290PSIyLjE2Ljg0MC4xLjExMzg4My4xMC4yMC4yMi40LjI3IiBleHRlbnNpb249IjIwMTQtMDYtMDkiLz4NCgkJCQkJCQkJCTxpZCByb290PSI0Y2U2MDQ2Yy1mNmUzLTQxYjAtOTFmYy0yZDUzMjVmMmJiYzMiLz4NCgkJCQkJCQkJCTxjb2RlIGNvZGU9IjI3MTAtMiIgY29kZVN5c3RlbT0iMi4xNi44NDAuMS4xMTM4ODMuNi4xIiBjb2RlU3lzdGVtTmFtZT0iTE9JTkMiIGRpc3BsYXlOYW1lPSJPWFlHRU4gU0FUVVJBVElPTiIvPg0KCQkJCQkJCQkJPHRleHQ+DQoJCQkJCQkJCQkJPHJlZmVyZW5jZSB2YWx1ZT0iI1NQTzJfMSIvPg0KCQkJCQkJCQkJPC90ZXh0Pg0KCQkJCQkJCQkJPHN0YXR1c0NvZGUgY29kZT0iY29tcGxldGVkIi8+DQoJCQkJCQkJCQk8ZWZmZWN0aXZlVGltZSB2YWx1ZT0iMjAxNDEwMDExMDMwMjYtMDUwMCIvPg0KCQkJCQkJCQkJPHZhbHVlIHhzaTp0eXBlPSJQUSIgdmFsdWU9Ijk4IiB1bml0PSIlIi8+DQoJCQkJCQkJCQk8YXV0aG9yIHR5cGVDb2RlPSJBVVQiPg0KCQkJCQkJCQkJCTx0aW1lIHZhbHVlPSIyMDE0MTAwMTEwMzAyNi0wNTAwIi8+DQoJCQkJCQkJCQkJPGFzc2lnbmVkQXV0aG9yPg0KCQkJCQkJCQkJCQk8aWQgZXh0ZW5zaW9uPSI1NTU1NTU1NTU1IiByb290PSIxLjEuMS4xLjEuMS4xLjIiLz4NCgkJCQkJCQkJCQk8L2Fzc2lnbmVkQXV0aG9yPg0KCQkJCQkJCQkJPC9hdXRob3I+DQoJCQkJCQkJCTwvb2JzZXJ2YXRpb24+DQoJCQkJCQkJPC9jb21wb25lbnQ+DQoJCQkJCQk8L29yZ2FuaXplcj4NCgkJCQkJPC9lbnRyeT4NCgkJCQk8L3NlY3Rpb24+DQoJCQk8L2NvbXBvbmVudD4NCgkJPC9zdHJ1Y3R1cmVkQm9keT4NCgk8L2NvbXBvbmVudD4NCjwvQ2xpbmljYWxEb2N1bWVudD4="
```

Comment:
I know I approved this, but I didn't see this file as GH didn't show it bc it was too large.

Can we store this on a file? We have examples of file system usage with `fs` on this repo.

Reply:
put this in a file